### PR TITLE
Migrate Prometheus Module to Package

### PR DESCRIPTION
**Pull request by ChrsMark**
Implementing Prometheus package migration

Screenshots:
Screenshot 2020-06-29 at 20 20 32
Screenshot 2020-06-29 at 20 22 54
Screenshot 2020-06-29 at 20 22 28
Screenshot 2020-07-06 at 12 37 16

### CODE REVIEW

**Comment by mtojek** [comment]
Please also rebase against master.

**Comment by ChrsMark** [comment]
Thanks for the review @mtojek ! 

PR has been updated:

- Added screenshot
- Addressed comments and rebased
- Renamed `datasources` to `config_templates` 
- Renamed `metrics` category to `datastores`

File: packages/prometheus/dataset/collector/fields/fields.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+- name: prometheus
```
Comment (exekias) [inline packages/prometheus/dataset/collector/fields/fields.yml:1]: This file seems to be missing some fields. I would expect it to be a mix between x-pack and oss fields.yml's, as we are shipping the basic agents with the agent

Comment (ChrsMark) [reply packages/prometheus/dataset/collector/fields/fields.yml:1]: 🤦 good catch, thanks!

**Comment by ChrsMark** [comment]
@exekias I added manually the missing fields and tested manually again to verify nothing is broken. Feel free to have another look :)!

**Pull request merged**